plugins/woocommerce/includes/wc-account-functions.php

```diff
@@ -164,6 +164,33 @@ function wc_get_account_menu_item_classes( $endpoint ) {
 	return implode( ' ', array_map( 'sanitize_html_class', $classes ) );
 }
 
+/**
+ * Get current page in order to set the current menu item.
+ *
+ * @param string $endpoint Endpoint.
+ * @return string
+ * @since 9.2.0
+ */
+function wc_get_current_page( $endpoint ) {
```

Comment:
I think this has the potential to be misleading, I'd recommend the following so it matches `wc_get_account_menu_item_classes` above.

```suggestion
function wc_get_account_menu_aria_current( $endpoint ) {
```

Do you agree?

Reply:
@opr agreed and updated 😄